template.yaml

```diff
@@ -742,6 +744,53 @@ Resources:
       LogGroupName: !Sub "/aws/lambda/${DashboardGetStudyPeriodsFunction}"
       RetentionInDays: !Ref RetentionTime
 
+  DashboardPostDistributeStudyFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub 'CumulusAggDashboardDistributeStudy-${DeployStage}'
+      PackageType: Image
+      ImageConfig:
+        Command: ["post_distribute.distribute_handler"]
```

Comment:
This (and the contents of the meta fields below) are the bits that switch the lambda from the normal execution environment to a container-based execution.